index.bs

```diff
@@ -380,6 +380,19 @@ examples that violate the above rules are {{XMLHttpRequest}} and
 initialisms, even if they are repeated.
 </div>
 
+<h4 id="naming-unsafe" class="no-num no-toc">Warning about dangerous features</h4>
+
+Many specifications provide guarantees or invariants to developers who use them.
+For example,
+[CSP](https://w3c.github.io/webappsec-csp/)
+provides protection against certain types of content injection vulnerabilities.
+Such specifications may also provide features that weaken their own invariants
+(or those of other specifications),
+such as CSP's `unsafe-inline` keyword,
+which allows inline scripts, reducing CSP's own protections.
+It is useful to mark features that weaken the invariants provided to developers
```

Comment:
Can we pull this up to the top as a summary?

"Mark feature that weaken invariants provided to developers by naming them starting with "unsafe"".

Reply:
Done in bcf3fb3 (with a good bit of other resulting change).